nncf/definitions.py

```diff
@@ -14,3 +14,5 @@
 import os
 NNCF_PACKAGE_ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 HW_CONFIG_RELATIVE_DIR = "common/hardware/configs"
+NNCF_CI_ENV_VAR_NAME = "NNCF_CI"
+NNCF_DEV_ENV_VAR_NAME = "NNCF_DEV"
```

Comment:
It would be great to add a description for these environment variables.

Reply:
Done